modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/CliConfig.scala

```diff
@@ -37,7 +38,6 @@ import generated.ProjectMetadata
  * @param target decoded target to load
  * @param steps collected steps
  * @param logKey file on S3 to dump logs
- * @param folder specific run-folder to load (skipping discovery)
```

Comment:
Don't we want to keep it until we drop batch support ?

Reply:
That one was another feature not relalted to the common way we use it, i.e. we could provide `--folder s3://bucket/some-path` to load data not from `shredded.good`. We introduced it ages ago for some complex recovery and I think nobody used it since then.